ui/app/src/components/PathNavigatorTree/PathNavigatorTree.tsx

```diff
@@ -439,6 +443,7 @@ export default function PathNavigatorTree(props: PathNavigatorTreeProps) {
     const anchorRect = element.getBoundingClientRect();
     const top = anchorRect.top + getOffsetTop(anchorRect, 'top');
     const left = anchorRect.left + getOffsetLeft(anchorRect, 'left');
+    dispatch(completeDetailedItem({ path }));
```

Comment:
Why? What is the detailed item needed just for the menu?

Reply:
the other pathNavigator is using it too, this is needed because: it refresh the item, so if the item has recently changes the AA are different, doing this the menu should have the recent changes if any, also is needed for a couple of dialogs. example: Publish/Approve Dialog